dwebp: add more examples (#6359)
tldr-pages/tldr: pages/common/dwebp.md

> Animated WebP files are not supported.
> More information: <https://developers.google.com/speed/webp/docs/dwebp/>.

- Convert a `webp` file into a `png` file:

`dwebp {{path/to/input.webp}} -o {{path/to/output.png}}`

- Convert a `webp` file into a specific filetype:

> `dwebp {{path/to/input.webp}} -bmp|-tiff|-pam|-ppm|-pgm|-yuv -o {{path/to/output}}`

Reviewer: Based on the `systemctl` page:
```suggestion
`dwebp {{path/to/input.webp}} -{{bmp|tiff|pam|ppm|pgm|yuv}} -o {{path/to/output}}`
```

Author: In systemctl we don't only have multiple keywords without a dash, so they're rather subcommands than flags. Also, when rendered I find the first one easier to understand:
![image](https://user-images.githubusercontent.com/50295997/129481930-48a9d146-b4f5-498f-bcc5-0f739c4737b3.png)

Reviewer 2: Let's keep this as is, but wrap in token syntax?
```suggestion
`dwebp {{path/to/input.webp}} {{-bmp|-tiff|-pam|-ppm|-pgm|-yuv}} -o {{path/to/output}}`
```

Author: Hmm. That would be a lot of token syntax and if you highlight almost everything it'l like highlighting nothing:
![image](https://user-images.githubusercontent.com/50295997/129484162-edad7d3c-cb3c-444e-8fdb-be30d9cc0123.png)

Reviewer 2: True. Let's leave it as-is then.

- Convert a `webp` file, using multi-threading if possible:

`dwebp {{path/to/input.webp}} -o {{path/to/output.png}} -mt`

> - Convert a `webp` file and crop and scale the output:

Reviewer 3: ```suggestion
- Convert a `webp` file, crop, and scale the output:
```

Author: I think since `crop` and `scale` both refer to `output`, they should be grouped extra. The sentence sounds a bit weird when just listing everything.

Reviewer 2: What about this?
```suggestion
- Convert a `webp` file, but also crop and scale at the same time:
```
